### DISCUSSION TITLE
Delay Statistics

### DISCUSSION THREAD

**Discussion by Omit6000** [Traewelling/traewelling#2021] [Ideas / Feature Requests]
List and add up delays in transport and display them as statistics.  So that you can see how much delay you have accumulated in a certain time interval.

**Comment by MrKrisKrisu** [comment]
It should be noted that a delay occurring during the journey is not currently saved. Only the delay data at the time of check-in is stored.
Therefore, such statistics would not (**currently**) make sense.

**Comment by Omit6000** [comment]
Hey, maybe you can do a synchronisation with "Travelynx" for delays.
Travelynx saves this data.
Greetings, Timo

Kris ***@***.***> schrieb am So., 14. Aug. 2022, 17:52:

**Comment by MrKrisKrisu** [comment]
We also save the delay. However, we do not currently update it **during the trip** because the associated script has been disabled.

see #1015

**Comment by ChristianTacke** [comment]
What about updating the delays if a user modifies the status?

**Comment by pascalwittler** [comment]
Meanwhile, Träwelling seems to update delays during a trip from time to time. At least updated estimated arrival times are visible and change several times (if necessary) in the status messages of still ongoing trips in the frontend:

![Screenshot](https://github.com/Traewelling/traewelling/assets/20669380/d164a251-e049-4346-ac06-c427b836ccdf)

(Last text line at the bottom of the screenshot: "Real time data updated 6 minutes ago")

**Comment by pascalwittler** [comment]
possible overlap with #2051

**Comment by rugk** [comment]
I agree this (or https://github.com/Traewelling/traewelling/discussions/2051) would be very helpful for enforcing passenger rights. /cc @ErikUden 

Here is how Travelyx saves the data and exposes it (it even has an export for the filled in form): https://travelynx.de/fgr
![grafik](https://github.com/user-attachments/assets/5993181c-6a6a-4f74-b2e5-4cd140c11d43)

Such a statistic would be very helpful.
